core/utils/checksum.h

```diff
@@ -74,6 +74,7 @@ static inline uint32_t CalculateSum(const void *buf, size_t len) {
   }
 #endif
 
+#if __x86_64
```

Comment:
Since this code block doesn't have anything specific to x86, can we use something else (e.g., `__LP64__`)..? While BESS only supports i686 and x86_64, it would be good to keep the code architecture neutral as much as we can.